packages/replay_bloc/example/lib/main.dart

```diff
@@ -3,12 +3,14 @@ import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:replay_bloc/replay_bloc.dart';
 
 void main() {
-  Bloc.observer = AppBlocObserver();
+  Bloc.observer = const AppBlocObserver();
   runApp(const App());
 }
 
 /// Custom [BlocObserver] that observes all bloc and cubit state changes.
 class AppBlocObserver extends BlocObserver {
+  /// Custom [BlocObserver] that observes all bloc and cubit state changes.
```

Comment:
Can we create a template and use a macro here to avoid duplicating the documentation?